packages/manager/apps/pci-workflow/src/pages/list/List.page.tsx

```diff
@@ -0,0 +1,361 @@
+  const { t: tFilter } = useTranslation('filter');
+  const projectUrl = useProjectUrl('pci-workflow');
+
+  const { data: workflows, isPending } = usePaginatedWorkflows(
+    projectId,
+    pagination,
+    filters,
+  );
+
+  const columns: DatagridColumn<TWorkflow>[] = [
+    {
+      id: 'name',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {workflow.name}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_name'),
+    },
+    {
+      id: 'id',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {workflow.id}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_id'),
+    },
+    {
+      id: 'type',
+      cell: () => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {t('pci_workflow_type_instance_backup_title')}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_type'),
+    },
+    {
+      id: 'resource',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsLink
+            color={ODS_THEME_COLOR_INTENT.primary}
+            href={`${projectUrl}/instances/${workflow.instanceId}`}
+          >
+            {workflow.instanceName}
+          </OsdsLink>
+        </>
+      ),
+      label: t('pci_workflow_resource'),
+    },
+    {
+      id: 'schedule',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {workflow.cron}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_schedule'),
+    },
+    {
+      id: 'lastExecution',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsText
+            level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            color={ODS_THEME_COLOR_INTENT.text}
+            className="block"
+          >
+            {workflow.lastExecution}
+          </OsdsText>
+        </>
+      ),
+      label: t('pci_workflow_last_execution'),
+    },
+    {
+      id: 'lastExecutionStatus',
+      cell: (workflow: TWorkflow) => (
+        <>
+          <OsdsChip
+            color={(() => {
+              switch (workflow.lastExecutionStatus) {
```

Comment:
Maybe create a separate function for it, as I'll need it in the executions listing.